manifests/w/workman-layout/workman/2.0.0/workman-layout.workman.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: workman-layout.workman
+PackageVersion: 2.0.0
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://github.com/joel0/Workman/raw/installer-repackage/windows/installer/workman-us/wm-us_amd64.msi

```

Comment:
```suggestion
  InstallerUrl: https://github.com/workman-layout/Workman/raw/master/windows/installer/workman-us/wm-us_amd64.msi
```

Reply:
Updated.  I used the commit hash in the URL to pin the version in case the file path changes in the future on the `master` branch.